### ISSUE TITLE
Feature Glacier Support

### ISSUE THREAD

**Issue by Fyb3roptik** [mmontagna/zfs3backup#2]
Looks like it was added here by a PR https://github.com/presslabs/z3/pull/19

Crossing my fingers this can happen!

**Comment by mmontagna** [comment]
Why glacier specifically? Glacier is some what of a pain to work with but reduced redundancy or infrequent access s3 objects are somewhat easier.

**Comment by Fyb3roptik** [comment]
Glacier is so much cheaper than S3

**Comment by mmontagna** [comment]
Yeah as long as you never want to access your data. Happy to accept a PR for that, but short of that I'd recommend using a lifecycle policy on your s3 bucket to convert the objects there to glacier storage after a period of time.

**Comment by Fyb3roptik** [comment]
I am using this to store my zfs snapshots of my plex server, so I dont NEED access to the data. Its only for backups
